yellowbrick/base.py

```diff
@@ -181,7 +181,7 @@ def finalize(self, **kwargs):
         """
         return self.ax
 
-    def poof(self, outpath=None, **kwargs):
+    def poof(self, outpath=None, clear_figure=False, **kwargs):
```

Comment:
Awesome thank you so much! Would you mind adding this to the `poof` docstring under parameters? It might be nice to explain why you might want to set this to True (e.g. explain the issue you had.

Reply:
Thanks for the reminder. Always forget docstring..... >.<